packages/terra-toolkit-docs/src/terra-dev-site/tool/terra-functional-testing/About.1.tool.mdx

```diff
@@ -64,59 +108,157 @@ Tests can be executed in any of the following supported form factors:
 
-All tests within this helper will be executed for each given viewport(s). Only the above defined viewports are supported. The default viewport is 'huge'. If a form factor is passed as a CLI option to the `test wdio` test runner, this form factor will supersede the viewports defined in this helper. This helper accepts these arguments in the following order:
-- {string} title - The `describe` block title.
-- {string[]} viewports - The list of Terra viewports to test.
-- {function} - the test function to execute for each viewport.
+To extend the default configuration create a `wdio.conf.js` file at the root of your project and apply the desired options.
 
+wdio.conf.js
 ```js
-Terra.describeViewports('Describe viewports', ['tiny', 'huge'], () => {
-  it('should test tiny and huge screens', () => {
-    browser.url('/testing/route/');
-    Terra.validates.screenshot('describe viewports');
-  });
-});
+const { config } = require('@cerner/terra-functional-testing');
+
+// Stop the test run if there is a test failure.
+config.bail = 1;
+
+exports.config = config;
 ```
 
-### Terra.describeTests
+## Service Options
+
+Service options are configurable by extending the wdio.conf.js file. These options are applied for every test run.
+
+### selector
+
+Specifies the default element to be captured when taking a screenshot.
+
+Type: `string`
 
-The `Terra.describeTests` helper extends the `Terra.describeViewports` helper by additionally filter tests by locales and themes. All tests within this helper will be executed only if each of the form factors, locales, and themes listed in the options match those defined in the configurations for the current test run. If no form factors, locales, or themes are provided as options, then all tests under this helper will qualify to run in any form factor, locale, or theme. It is recommended to use `Terra.describeTests` over `Terra.describeViewports` because the same features in `Terra.describeViewports` can be achieved with `Terra.describeTests` by eliminating locales and themes and only specify the form factors. This helper accepts these arguments in the following order:
-- {string} title - The describe block title.
-- {Object} options - An object containing arrays of formFactors, locales, and themes that the block of tests will only qualify to execute in.
-- {function} - The block of tests to execute against each viewport.
+Required: `false`
 
+Default: `[data-terra-test-content] *:first-child`
 
-The below describe block qualifies to run because each of the form factors, locales, and themes provided to `describeTests` match the CLI options specified in the test runner.
-```sh
-terra wdio --formFactors tiny --locales en --themes terra-default-theme
+<Notice variant="important" ariaLevel="3">
```

Comment:
i think this should be an arialLevel 4

Reply:
Should the aria level always be one level higher than the block it's inside of?

Reply:
yeah it's basically the heading level. if the notice is a child of the block it's inside of it should be a level down, if it's a peer then it should be on the same level.

Reply:
Bumped each of these based on the level of block they were inside:

https://github.com/cerner/terra-toolkit/pull/610/commits/e6ee8b1b58f5506e29afee2ba4159e16b4a082c8